samples/nrf9160/https_client/src/main.c

```diff
@@ -100,6 +100,9 @@ void pdn_event_handler(uint8_t cid, enum pdn_event event, int reason)
 	case PDN_EVENT_DEACTIVATED:
 		printk("PDP context %d deactivated\n", cid);
 		break;
+	case PDN_EVENT_NETWORK_DETACH:
+		printk("PDP context %d network detached\n", cid);
```

Comment:
@stig-bjorlykke should be log the `cid` here?

Reply:
The `cid` is not included in the AT notification, but it is included in the event sent to all pdn contexts.
This is ok, to indicate the context in use for the https_client.